### PR TITLE
Elasticsearch v2: fixed incident labels representation

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/33177

## Description
fixed incident labels representation.
in the case of JSON values, we returned `{'is_ok': True}` instead of `{"is_ok": true}`.

## Screenshots
screenshots with the fix posted in the issue.

### CODE REVIEW

File: Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py [hunk 1]
```diff
@@ -464,8 +464,9 @@ def incident_label_maker(source):
         (list).The labels.
     """
     labels = []
-    for field in source.keys():
-        labels.append({'type': str(field), 'value': str(source.get(field))})
+    for field, value in source.items():
+        encoded_value = value if isinstance(value, str) else json.dumps(value)
```
Comment (Itay4) [inline Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py:468]: can we add test ?

File: Packs/Elasticsearch/ReleaseNotes/1_1_4.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Elasticsearch v2
+- Fixed an issue where JSON incident labels were not saved properly.
```
Comment (Itay4) [inline Packs/Elasticsearch/ReleaseNotes/1_1_4.md:4]: i would elaborate on the change in rn as this theoretically can break bc

**Review by reutshal — COMMENTED** [review]
Add more tests.

File: Packs/Elasticsearch/ReleaseNotes/1_1_4.md (hunk 2 above)
Comment (yaakovi) [reply Packs/Elasticsearch/ReleaseNotes/1_1_4.md:4]: as most of our bugs, we used to have bad behavior, and now it was fixed.
simple object behavior did not change only complex objects.

**Review by reutshal — APPROVED** [review]

**Pull request merged**